doc/develop/sca/eclair.rst

```diff
@@ -0,0 +1,93 @@
+.. _eclair:
+
+ECLAIR support
+##############
+
+Bugseng `ECLAIR <https://www.bugseng.com/eclair/>`__ is a certified
+static analysis tool and platform for software verification.
+Applications range from coding rule validation, with a
+particular emphasis on the MISRA and BARR-C coding standards, to the
+computation of software metrics, to the checking of independence and
+freedom from interference among software components, to the automatic
+detection of important classes of software errors.
+
+.. important::
+
+   ECLAIR is a commercial tool, and it is not free software.
+   You need to have a valid license to use it.
```

Comment:
Does this mean that the only way a contributor can respond to flagged issues in their PR is to try to fix it then resubmit to Zephyr CI for another pass? I say this because it would be nice if there was a path for local support without having to burden the Zephyr compute resources in a PR.

Reply:
> Does this mean that the only way a contributor can respond to flagged issues in their PR is to try to fix it then resubmit to Zephyr CI for another pass? I say this because it would be nice if there was a path for local support without having to burden the Zephyr compute resources in a PR.

actually for this first integration yes because we need to get a feel for the tool and how we can handle it in the environment.
But the next step would be to integrate it into the CI. Bugseng already have a solution for this.
A demo how it would look like in a github PR can be found here: https://github.com/BUGSENG/eclair_demo/pull/14